.eslintrc

```diff
@@ -1,6 +1,14 @@
 {
   "extends": "eslint-config-mitodl",
   rules: {
-      "no-unused-vars": 0
+    "no-unused-vars": 0,
+    "no-restricted-imports": [
```

Comment:
This is just to restrict future import style.

Reply:
Can you include a comment in this file to explain?